GPU/GLES/FramebufferManagerGLES.cpp

```diff
@@ -1083,14 +1005,14 @@ void FramebufferManagerGLES::PackFramebufferSync_(VirtualFramebuffer *vfb, int x
 
 	if (packed) {
 		DEBUG_LOG(FRAMEBUF, "Reading framebuffer to mem, bufSize = %u, fb_address = %08x", bufSize, fb_address);
-
-		SafeGLReadPixels(0, y, h == 1 ? packWidth : vfb->fb_stride, h, GL_RGBA, GL_UNSIGNED_BYTE, packed);
-
+		int packW = h == 1 ? packWidth : vfb->fb_stride;  // TODO: What's this about?
```

Comment:
I *think* this was part of a workaround for issues with GL_PACK_ROW_WIDTH or whatever not working, probably should have a comment. Or it might have just been fear of breaking it again when adding the ability to read less than stride for palettes.

Ideally it should not use stride, in case there's data in the "holes" as some games (like MotoGP) use them for textures.  I think we already use a temp buffer maybe for that case?

-[Unknown]